packages/twenty-front/src/modules/object-metadata/utils/mapPaginatedObjectMetadataItemsToObjectMetadataItems.ts

```diff
@@ -11,6 +10,8 @@ export const mapPaginatedObjectMetadataItemsToObjectMetadataItems = ({
     pagedObjectMetadataItems?.objects.edges.map((object) => ({
```

Comment:
if (!isString(object.node
        .labelIdentifierFieldMetadataId) { throw / continue }

Reply:
put in place zod validation schema inferred from the `objectMetadataItem` zod object